lib/meshchat/configuration/settings.rb

```diff
@@ -60,19 +67,12 @@ def generate_uid
       end
 
       def generate_keys
-        @key_pair = OpenSSL::PKey::RSA.new(2048)
-        self['publickey'] = @key_pair.public_key.to_pem
-        self['privatekey'] = @key_pair.to_pem
+        pubilc_key, private_key = Encryption.generate_keys
+        self['publickey'] = to_base64(pubilc_key)
+        self['privatekey'] = to_base64(private_key)
```

Comment:
@etkirsch, @ecollis6, I think we should add to the mesh-chat-protocol, that all text versions of keys should be base64